components/tor/tor_control.h

```diff
@@ -18,6 +18,7 @@
 #include "base/callback.h"
 #include "base/files/file_path.h"
 #include "base/memory/scoped_refptr.h"
+#include "base/memory/weak_ptr.h"
 #include "base/observer_list.h"
```

Comment:
seems unused, please double-check for unused headers

Reply:
fixed in aeed04b506e574973c5df12c9781dde63e187342